force-app/main/default/objects/ProgramEngagement__c/fields/Stage__c.field-meta.xml

```diff
@@ -22,6 +22,11 @@
                 <default>false</default>
                 <label>Application Denied</label>
             </value>
+            <value>
+                <fullName>Waitlisted</fullName>
```

Comment:
EDA seems to use the stage Waitlist as opposed to Waitlisted. I'm checking with Docs to see if that's their preference.

Reply:
Docs thinks Waitlisted is most consistent with our other stages. We are good to go.